src/pages/components/data-table/usage.mdx

```diff
@@ -277,148 +280,312 @@ import {
+<Row>
+<Column colLg={8}>
 
-<div className="image--fixed">
+![data table size](images/datatable-sizes.png)
 
-![data table with search](images/data-table-usage-3.png)
+</Column>
+</Row>
 
-</div>
+The column header row `.bx--data-table thead` should always match the row size of the table. Tall row heights are only recommended if your data is expected to have 2 lines of content in a single row.
 
-<Caption>Search on data tables.</Caption>
+<DoDontRow>
+<DoDont
+colMd={4} colLg={4}
+caption="Do use the same row height for the table and header rows.">
 
-### Sorting
+![Do: uDo use the same row height for the table and header rows.](./images/datatable-header-size-do.png)
 
-Make data sortable to improve table usability. Sorting controls are located in the column headers and indicated with an arrow icon.
+</DoDont>
+<DoDont
+colMd={4} colLg={4}
+type="dont"
+caption="Don’t mix row heights for in the table and header rows.">
 
-A sorted data table has three states: unsorted (`arrows`), sorted-up (`arrow--up`) or sorted-down (`arrow--down`). The icon indicates the current sorted state and is only shown if sorting is activated. Only the column being sorted should display an icon, and unsorted icons are only visible on hover. You can see a demo of table sorting in Carbon’s [React Storybook](http://react.carbondesignsystem.com/?path=/story/datatable--with-sorting).
+![Don’t: mix row heights for in the table and header rows.](./images/datatable-header-size-dont.png)
 
-<div className="image--fixed">
+</DoDont>
+</DoDontRow>
 
-![Data table with sorting](images/data-table-usage-13.png)
+### Placement
 
-</div>
+Data tables should be placed in a page's main content area and given plenty of space to display data without truncation. Avoid placing data tables inside modals or smaller containers where the information can feel cramped or needs truncation.
 
-<Caption>Sorting on data tables</Caption>
+The data table can be placed on the grid following the three different [grid modes](/guidelines/2x-grid/implementation#grid-modes) outlined in the 2x grid guidelines. Although, the data table can share horizontal space with other components and content, consider giving your data table the most width on the page to help your user view dense data.
 
-### Table toolbar
+These three examples show the data table on the wide (default), narrow, and condensed grid modes. Note the different type alignments and gutter sizes.
 
-The table toolbar is reserved for global table actions such as table settings, complex filter, export, or editing table data. Additional options can be added if you have an icon to support that function. Icons should be 16x16 px (glyphs). We recommend using no more than 5 icons within the table toolbar.
+#### Wide grid mode
 
-<div className="image--fixed">
+The wide grid mode is the default and gives the most breathing room between the data table and the other components or content.
 
-![Table toolbar](images/data-table-usage-6.png)
```

Comment:
It would be good to mention somewhere (maybe the sizing section?) that the toolbar and the batch actions bar come in two different sizes depending on which height of data table you are using. Jan might have recommendations on where to put this content.